OpenRA.Mods.Common/Widgets/SliderWidget.cs

```diff
@@ -103,7 +117,7 @@ Rectangle ThumbRect
 				var thumbPos = PxFromValue(Value);
 				var rb = RenderBounds;
 				var width = rb.Height;
-				var height = rb.Height;
+				var height = width;
```

Comment:
This looks really weird. Are we sure this is intentional? (Perhaps get a comment.)